### PR TITLE
v0.16 release prep

### LINKED ISSUE #125: Release prep cleanup
In preparation for `v0.16`

- [x] Close PR #119
- [x] Close PR #117
- [x] Convert readme to RST
- [x] Convert changelog to Markdown
- [x] Move `VERSION` -> `ais/__init__.py`
- [x] Move `ais_lut.py` -> `ais/enums|lut.py` (or something similar?  looks like they're not in use)
- [x] Move `maritime_id_codes.py` -> `utils`
- [x] Put `Apache 2.0` in `ais.__license__`
- [x] Remove `libais_decode` from `console_scripts` in `setup.py` - doesn't exist
- [x] Delete `nagios_pg_ais.py`
- [x] Delete `nais2pg.py`
- [x] Delete `create.sql`
- [x] Delete `TODO.org` (unlessn needed)
- [x] Add deprecation warning to `bin/aisdecode`
- [x] Add deprecation warning to `ais.stream` and `ais.compatibility`
- [x] Update readme with `ais.open()` example (After #117 is closed)
- [x] Update `MANIFEST.in`
- [x] Update `setup.py`

### PR DESCRIPTION
**Pull request by geowurster**
Closes #125 

@schwehr Ready for initial review but needs to be merged after #117 so I can add `ais.open()` to the readme and resolve some merge conflicts.

### CODE REVIEW

**Comment by geowurster** [comment]
@schwehr Just found your comments on my [gist](https://gist.github.com/geowurster/f97981ee9c6799652aa5).  The benefit to switching the readme to RST is that PyPi will render it if you put the content in the `long_description` argument in `setup.py`, which I just noticed I neglected to do.  Will fix that.

I'll defer to you on the final fate of `ais_lut.py`.  I moved it to `ais/ais_lut.py` in this PR.

File: bin/aisdecode [hunk 1]
```diff
@@ -37,7 +50,7 @@ for key in ("uscg", "validate_checksum", "verbose", "allow_unknown", "treat_ab_e
         args[key] = args[key] == 'True'
 
 if "help" in args:
-    print """Usage: aisdecode [OPTIONS] < INFILE.nmea > OUTFILE.json
+    print("""Usage: aisdecode [OPTIONS] < INFILE.nmea > OUTFILE.json
```
Comment (schwehr) [inline bin/aisdecode]: Shouldn't this mean that there needs to be a from future for print for py 2.7?

**Comment by schwehr** [comment]
Needs a tweak to Makefile-custom to get the version from ais/__init__py.

File: bin/aisdecode (hunk 1 above)
Comment (geowurster) [inline bin/aisdecode]: @schwehr No - a lot of the functionality in `__future__` was backported.  This works for `2.6` as well.

**Comment by geowurster** [comment]
@schwehr Updated `Makefile-custom`.  Is `xargs` a portable command for *nix systems?  It's converting ` '0.16'` to `0.16` by stripping whitespace and quotes.

File: Makefile-custom [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # -*- makefile -*-
 
-VERSION:=${shell cat VERSION}
+VERSION:=${shell cat ais/__init__.py | grep __version__ | cut -d = -f 2 | xargs}
```
Comment (schwehr) [inline Makefile-custom]: can you explain this use of xargs with no args?

Comment (geowurster) [inline Makefile-custom]: Did some more research and looks like this isn't a good idea.  Just pushed an alternative.

Comment (schwehr) [inline Makefile-custom]: What about this"

    echo "__version__ = '0.16'" | cut -d\' -f 2
    0.16

Comment (geowurster) [inline Makefile-custom]: That assumes the quotes will always be single.

**Pull request merged**